pricefeed-golang/README.md

```diff
@@ -0,0 +1,65 @@
+# Getting an asset price from EVM chain
+
+## Table of Contents
+
+- [About The Project](#about-the-project)
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+- [Usage](#usage)
+
+## About The Project
+
+The project shows how to retrieve asset price from any EVM chain. The contract addresses can be found [here](https://docs.chain.link/docs/reference-contracts/).
```

Comment:
recommend we make more specific by saying "The <<data feed>> contract addresses.  For beginners it is not obvious what smart contracts are involved.

Reply:
ok good point. will do it . thanks